xml/System.Windows.Forms/ToolStripItem.xml

```diff
@@ -7273,8 +7278,12 @@ If you set the <xref:System.Windows.Forms.ToolStripItem.ImageKey%2A> property, t
         <ReturnType>System.EventHandler</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Occurs when value of the <see cref="P:System.Windows.Forms.ToolStripItem.Selected" /> property changes.</summary>
+        <remarks>
+          <para>This event is raised when item is selected by mouse, keyboard or programmatically.</para>
+          <para>.NET Core 3.1 had removed MainMenu and MenuItem controls. ToolStripMenuItem.SelectedChanged event</para>
+          <para>is recommended as a replacement for MenuItem.Select event.</para>
```

Comment:
Would you mind reinstalling the porting tool so that this does not happen again? Simply sync to the latest bits in main and run the `.\install-as-tool.ps1` from dotnet/api-docs-sync. I fixed the annoying `<para>` bug yesterday.

Also, some fixes here. I think ToolStrip**Menu**item might be wrong in the text, can you confirm?:

```suggestion
          <para>This event is raised when item is selected by mouse, keyboard or programmatically.</para>
          <para>The <c>MainMenu</c> and <c>MenuItem</c> controls were removed in .NET Core 3.1. The <see cref="E:System.Windows.Forms.ToolStripItem.SelectedChanged" /> event is recommended as a replacement for the <c>MenuItem.Select</c> event.</para>
```

Reply:
I believe this was intentional to be ToolStripMenuItem.SelectedChanged @Tanya-Solyanik can you double check if that is correct?